docs/molgenis/use_usingpyclient.md

```diff
@@ -0,0 +1,281 @@
+| argument | type | description                               | required | default |
+|----------|------|-------------------------------------------|----------|---------|
+| table    | str  | the name of a table                       | False    | None    |
+| schema   | str  | the name of a schema                      | False    | None    |
+| fmt      | str  | the output format, either `csv` or `xlsx` | False    | `csv`   |
+
+
+### save_schema
+```python
+client.save_schema(table='Data Table', name='My Schema',  
+                   file='location/of/data/file.csv', data=[{'col1': value1, ...}, {'col2': value2, ...}, ...])
+```
+Imports or updates records in a table of a named schema.
+The data either originates from a file on the disk, or is supplied by the user after, for example, preprocessing.
+Either `file` or `data` must be supplied. The data must be compatible with the schema to which it is uploaded. 
+Throws the `PermissionDeniedException` if the user does not have at least _editor_ permissions for this schema.
+Throws the `NoSuchSchemaException` if the schema is not found on the server.
+
+| argument | type | description                            | required | default |
+|----------|------|----------------------------------------|----------|---------|
+| table    | str  | the name of a table                    | True     |         |
+| schema   | str  | the name of a schema                   | False    | None    |
+| file     | str  | the location of a `csv` file with data | False    | None    |
+| data     | list | data as a list of dictionaries         | False    | None    |
+
+
+### delete_records
+```python
+client.delete_records(table='Data Table', name='My Schema',  
+                      file='location/of/data/file.csv', data=[{'col1': value1, ...}, {'col2': value2, ...}, ...])
+```
+Deletes data records from a table.
+As in the `save_schema` method, the data either originates from disk or the program.
+Throws the `PermissionDeniedException` if the user does not have at least _editor_ permissions.
+Throws the `NoSuchSchemaException` if the schema is not found on the server.
+
+[//]: # (&#40;In order to delete records from a table the data must specify the row values with principal keys)
+
+| argument | type | description                            | required | default |
+|----------|------|----------------------------------------|----------|---------|
+| table    | str  | the name of a table                    | True     |         |
+| schema   | str  | the name of a schema                   | False    | None    |
+| file     | str  | the location of a `csv` file with data | False    | None    |
+| data     | list | data as a list of dictionaries         | False    | None    |
+
+
+### create_schema
+```python
+client.create_schema(name='New Schema', description='My new schema!', template='DATA_CATALOGUE', include_demo_data=True)
+```
+Creates a new schema on the server.
+If no template is selected, an empty schema is created.
```

Comment:
Would like to have a link to the available templates but they seem to missing in the emx2 documentation.

Reply:
Indeed, there should be. I believe the templates are being shaken up, so they should receive some updated documentation as soon as that is finished